LBH.AdultSocialCare.Api/V1/Factories/ApiToDomainFactory.cs

```diff
@@ -18,7 +18,15 @@ public static void Configure(IMapper mapper)
 
         public static DayCarePackageForCreationDomain ToDomain(this DayCarePackageForCreationRequest dayCarePackageForCreation)
         {
-            return _mapper.Map<DayCarePackageForCreationDomain>(dayCarePackageForCreation);
+            var res = _mapper.Map<DayCarePackageForCreationDomain>(dayCarePackageForCreation);
+            // Set status to 1 for new package
+            if (res.StatusId == 0)
```

Comment:
I thought we will set statusId on Gateway. What is the best practice I'm not sure.

Reply:
Discuss